docs/guide/en/README.md

```diff
@@ -1,9 +1,9 @@
-# Documentation
+# Yii File Router
 
-> Note: The router can be used along with the [`yiisoft/router`](https://github.com/yiisoft/router) package.
-
-> Note: Once the router found a matching route, it will interrupt the middleware queue and execute the controller
-> action.
+> **Notes:**
+>
+>- The router can be used along with the [`yiisoft/router`](https://github.com/yiisoft/router) package.
+>- Once the router found a matching route, it will interrupt the middleware queue and execute the controller action.
```

Comment:
```suggestion
> Note: The router can be used along with the [`yiisoft/router`](https://github.com/yiisoft/router) package.
> Once the router found a matching route, it will interrupt the middleware queue and execute the controller action.
```